app/assets/stylesheets/modules/research-starter.scss

```diff
@@ -0,0 +1,72 @@
+.research-starter:before {
+  position: relative;
+  top: 10px;
+  left: 10px;
+  font-size: 16px;
+  font-weight: 400;
+  color: #8c1515;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  content: "Research Starter";
```

Comment:
This is how @jvine specified it. I assumed it was needed like this because it's in a `:before` block.